### PR TITLE
card changes

### PR DESCRIPTION
**Pull request by annielovesjs**
Changes:
- display version number and language type on cards 
- styling changes

How to test:
- should look like this:
pagesDesign2
pagesDesign

### CODE REVIEW

**Comment by piotrmark** [comment]
Maybe it would be more readable if we display version first and {xxx} framework below?
Right now it feels a little bit like version applies to JS / Python

File: src/client/src/translations/en.json [hunk 1]
```diff
@@ -140,7 +140,6 @@
   "selectPages.pagesTitleQuestion": "Select pages for your application.",
   "selectPages.webAppTitleQuestion": "Select a project type.",
   "selectableCard.details": "Details",
-  "selectableCard.selected": "Selected",
```
Comment (streetcoder123) [inline src/client/src/translations/en.json:146]: Is it not being used now? Did you run `npm run manage:translations`?

Comment (annielovesjs) [reply src/client/src/translations/en.json:146]: yeah I deleted the text

Comment (annielovesjs) [reply src/client/src/translations/en.json:146]: since it's not being used anywhere now

**Comment by annielovesjs** [comment]
> Maybe it would be more readable if we display version first and {xxx} framework below?
> Right now it feels a little bit like version applies to JS / Python

Good point! I just swapped the order

File: src/client/src/components/SelectableCard/index.tsx [hunk 2]
```diff
@@ -136,12 +142,6 @@ const SelectableCard = ({
           >
             {(isPagesSelection && <div>{clickCount}</div>) || (
               <div className={styles.selectedText}>
-                <div>
-                  <FormattedMessage
-                    id="selectableCard.selected"
-                    defaultMessage="Selected"
```
Comment (jasmingoh) [inline src/client/src/components/SelectableCard/index.tsx:144]: is removing the "selected" message ok from the accessibility standpoint? will screen reader knows which card is selected without the message?

Comment (annielovesjs) [reply src/client/src/components/SelectableCard/index.tsx:144]: The screen reader never picked up the message before I removed it. There are a lot of accessibility problems like this throughout the wizard which I will work on in another PR

**Review by jsondoo — APPROVED** [review]
looks good

**Comment by streetcoder123** [comment]
Discussed with @annielovesjs the cards on Azure services page are a little off - slightly bigger for Hosting and smaller for Database. That would be addressed in a separate PR.

**Review by streetcoder123 — APPROVED** [review]

File: src/client/src/components/CardBody/index.tsx [hunk 3]
```diff
@@ -5,15 +5,17 @@ import styles from "./styles.module.css";
...
 
 type Props = InjectedIntlProps & IProps;
 
-const CardBody = ({ body, formattedBody, intl }: Props) => {
+const CardBody = ({ body, version, formattedBody, intl }: Props) => {
   return (
     <div className={styles.body}>
-      {body || (formattedBody && intl.formatMessage(formattedBody))}
+      {version && <div>v{version}</div>}
+      {body || (formattedBody && intl.formatMessage(formattedBody))}{" "}
```
Comment (piotrmark) [inline src/client/src/components/CardBody/index.tsx:18]: Do we need this space at the end?

File: src/client/src/components/CardBody/index.tsx [hunk 4]
```diff
@@ -5,15 +5,17 @@ import styles from "./styles.module.css";
...
 interface IProps {
   body?: string;
+  version?: string;
   formattedBody?: FormattedMessage.MessageDescriptor;
 }
 
 type Props = InjectedIntlProps & IProps;
 
-const CardBody = ({ body, formattedBody, intl }: Props) => {
+const CardBody = ({ body, version, formattedBody, intl }: Props) => {
```
Comment (piotrmark) [inline src/client/src/components/CardBody/index.tsx:14]: I know you didn't change this, but I find this logic with body and formattedBody super confusing. Maybe there is a way to simplify it?

File: src/client/src/components/SelectableCard/index.tsx [hunk 5]
```diff
@@ -102,7 +104,11 @@ const SelectableCard = ({
         )}
         <div className={grid.row}>
           <div className={styles.body}>
-            <CardBody body={body} />
+            {version ? (
```
Comment (piotrmark) [inline src/client/src/components/SelectableCard/index.tsx:108]: Feels like we don't need this condition here as CardBody already checks: {version && ... }

**Review by jcoc611-microsoft — APPROVED** [review]

**Pull request merged**